app/controllers/admin/unsynced_applications_controller.rb

```diff
@@ -5,13 +5,9 @@ def index
     @pagy, @applications = pagy(scope)
   end
 
-  def show
-    @application = Application.unsynced.includes(:user).find(params[:id])
-  end
```

Comment:
This is merged into admin/applications#show.

The pages were identical and merging them together simplifies the code.